util/elementsd_connector.go

```diff
@@ -31,7 +31,34 @@ func ReissueAsset(reissueTx string) (txid string, err error) {
 	errstr := stderr.String()
 
 	if err != nil || len(errstr) > 0 {
-		err = errors.New("reissuance of RDDL failed")
+		err = errors.New("reissuance of RDDL failed: " + errstr)
+	} else {
+		var txobj ReissueResult
+		err = json.Unmarshal(stdout.Bytes(), &txobj)
+		if err == nil {
+			txid = txobj.Txid
+		}
+	}
+	return txid, err
+}
+
+func DistributeAsset(address string, amount string) (txid string, err error) {
+	conf := config.GetConfig()
+	cmd := exec.Command("/usr/local/bin/elements-cli", "-rpcpassword="+conf.RPCPassword,
```

Comment:
Please use proper elements RPC endpoint. We already try to get rid of `planetmint-god tx ...` calls

Reply:
I want to avoid mixing up stuff and focusing on the distribution & pop integration. After that, I will start to integrate the [RPC library. ](https://github.com/rddl-network/elements-rpc)